app/destination/page.js

```diff
@@ -1,114 +1,68 @@
-              name="europa"
-              onRemove={() => removeFromWishlist('europa')}
-              thumbnail="/destination/image-europa.png"
-            />
-            <PlanetWishlistItem 
-              name="europa"
-              onRemove={() => removeFromWishlist('europa')}
-              thumbnail="/destination/image-europa.png"
-            />
-          </div> */}
+            {selectedPlanets.map((planet, index) => (
+              <PlanetWishlistItem
+                key={index}
+                name={planet.name}
+                onRemove={() => setSelectedPlanets(prev => prev.filter(p => p.name !== planet.name))}
+                thumbnail={planet.thumbnail}
+              />
+            ))}
+          </div>
         </section>
+
         <section className="card">
           <h2>Possible destinations</h2>
-          {/* TASK - React 1 week 2 */}
-          {/* Add all 4 planets! Europa, Moon, Mars, Titan  */}
-          {/* Use the README.md file for descriptions */}
-          {/* Create a <PlanetCard /> component, which accepts the following properties: */}
-          {/* name, description, thumbnail, isSelected, onAddOrRemovePlanet */}
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
+          {['Europa', 'Moon', 'Mars', 'Titan'].map((planetName) => {
+            const thumbnail = `/destination/image-${planetName.toLowerCase()}.png`;
+            const isSelected = selectedPlanets.some(p => p.name === planetName);
+
+            return (
+              <div className={styles.planetCard} key={planetName}>
+                <img className={styles.planetThumbnail} src={thumbnail} alt={planetName} />
+                <div className={styles.planetDescription}>
+                  <h2>{planetName} {isSelected ? "- SELECTED" : ""}</h2>
+                  <p>Lorem ipsum...</p>
+                </div>
+                <button 
```

Comment:
Even here, I would try to make it a little bit cleaner. Maybe store the planets value inside a variable and save this into a component so it is more readable and cleaner, and it'll be easier to maintain too!